### ISSUE TITLE
深度学习基础理论————学习率优化方法(AdaGrad/RMSprop/Adam/Warm-UP)-Big-Yellow-J

### ISSUE THREAD

**Issue by Big-Yellow-J** [Big-Yellow-J/discuss-store#4]
https://big-yellow-j.github.io/posts/2025/01/04/learning_rate.html 

 深度学习基础理论————学习率优化方法(AdaGrad/RMSprop/Adam/Warm-UP)
